### DISCUSSION TITLE
RTPS/DDSI submessage 0x80

### DISCUSSION THREAD

**Discussion by RowdenRichardConway** [eProsima/Fast-DDS#3921] [Q&A]
In wireshark (version 4.0.8) I can see that all of the RTPS packets from all of my FastDDS participants add an extra submessage to the end with a submessage ID of 0x80.  This is not part of the main DDSI specification so wireshark just labels it as Unknown/Vendor-Specific and makes no further effort to parse it.

I was wondering if anyone can explain, or point me towards documentation which explains, what this submessage is and what purpose it serves, along with hopefully any instructions on how to configure my FastDDS participants to not bother sending this submessage (As all the other DDS particpants in my deployment are from other vendors which I am assuming, like wireshark, also won't know how to parse this non-standard submessage)

**Comment by JLBuenoLopez** [comment]
@RowdenRichardConway 

That specific submessage is related to the [Fast DDS Statistics module](https://fast-dds.docs.eprosima.com/en/latest/fastdds/statistics/statistics.html#statistics-module). By default the module is built into Fast DDS, so you are going to capture those packages. In order to disable them, the only option is to build Fast DDS from sources disabling the module compilation using the CMake option `FASTDDS_STATISTICS` set to `OFF`. Nevertheless, other vendors would probably ignore a non-standard/vendor-specific submessage, so you do not have to worry about interoperability issues caused by this submessage.
